lib/xpi.js

```diff
@@ -48,7 +49,7 @@ function xpi(manifest, options) {
   }
   xpiName += ".xpi";
   var updateRdfName = getID(manifest) + xpiVersion + ".update.rdf";
-  var xpiPath = join(options.xpiPath || addonDir, xpiName);
+  var xpiPath = join(options.xpiPath || options.destDir || addonDir, xpiName);
   var updateRdfPath = join(options.xpiPath || addonDir, updateRdfName);
```

Comment:
Since `update.rdf` is also an output file, could we change that location as well please?